index.html

```diff
@@ -744,6 +787,46 @@ <h2>
             </p>
           </div>
         </section>
+        <section>
+          <h2><dfn>CaptureStartFocusBehavior</dfn></h2>
+          <p>
+            Describes whether an application invoking {{CaptureController/setFocusBehavior()}}
+            would like the user agent to focus the [=display surface=] associated with
+            that {{CaptureController}}'s [=capture-session=].
+          </p>
+          <pre class="idl">
+            enum CaptureStartFocusBehavior {
+              "focus-captured-surface",
+              "no-focus-change"
+            };    
+          </pre>
+          <table  data-dfn-for="CaptureStartFocusBehavior" class="simple">
+            <tbody>
+              <tr>
+                <th colspan="2">
+                  Enumeration description
+                </th>
+              </tr>
+              <tr>
+                <td>
```

Comment:
```suggestion
                <td nowrap>
```

After
![image](https://user-images.githubusercontent.com/634478/194589525-f6c7a426-93f1-486d-bf55-fa0664794c96.png)
Before
![image](https://user-images.githubusercontent.com/634478/194589625-5d876e09-4ee8-4e43-82bc-6b5bbf016c73.png)

Reply:
Actually, that seems to fail the bots.

Reply:
It says to use CSS. I'd suggest you replace `nowrap` with `style="white-space:nowrap"` to keep this behavior.

Reply:
That worked.